content/en/docs/tasks/observability/kiali/index.md

```diff
@@ -247,6 +247,40 @@ request traffic to route to two or more workloads.
 
     {{< image width="80%" link="./kiali-wiz4-ratings-weighted-route-90-10.png" caption="90% Ratings Traffic Routed to ratings-v2" >}}
 
+## Validating Istio configuration
+
+Kiali can validate your Istio resources to ensure they follow proper convention and semantics. Any problems detected in the configuration of your Istio resources can be flagged as errors or warnings depending on the severity of the misconfiguration. See the [Kiali validations page](http://kiali.io/documentation/validations/) for the list of all validation checks Kiali performs.
+
+Force a misconfiguration of a service port name to see how Kiali reports a validation error.
```

Comment:
How about adding:

{{< idea >}}
Istio 1.4 introduces [`istioctl analyze`](/docs/ops/diagnostic-tools/istioctl-analyze/) which lets you perform similar analysis in a way that can be used in a CI pipeline.
{{< /idea >}}

Reply:
done